### ISSUE TITLE
Swiping mechanism for VoiceOver navigation interferes with app's swiping mechanism

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1137]
### Description of problem
Swiping mechanism on the app interferes with the swiping mechanism of the OS's VoiceOver in the onboarding screens

### Expected behavior
Users should be able to swipe left or right to navigate to different elements on the screen without accidentally activating the swiping mechanism that switches screens on onboarding screens.

### Steps to reproduce
- Turn on VoiceOver on an iOS phone
- Go to onboarding screen in BC Wallet
- Attempt to swipe right to navigate to elements

### Screenshots and/or log output
N/A 

### Environment
- [ ] Occurs on Android
- [X] Occurs on iOS

Build #: 905

Android Device Model: 

iOS Device Model: iPhone SE

### Workaround
None

### Severity
- [ ] High
- [ ] Medium
- [X] Low

**Comment by knguyenBC** [comment]
Spoke with Bryce. As of now, the swiping mechanism only allows people to navigate to the skip button using Voice Over the first time they navigate pass it. Current fix is to give the skip button more context. Skip button to include alt-text "Skip introduction to BC Wallet" so people can understand what they're skipping.
